.vscode/extensions.json

```diff
@@ -0,0 +1,3 @@
+{
+  "recommendations": ["bradlc.vscode-tailwindcss"]
```

Comment:
We will want to add more to this, but this is fine to start with, at least we will want:

- dbaeumer.vscode-eslint
- esbenp.prettier-vscode

There will definitely be more, but these are probably the minimum.